app/src/main/res/layout/activity_sales_overview2.xml

```diff
@@ -63,8 +54,14 @@
         android:layout_marginEnd="@dimen/general_margin"
         app:layout_constraintBottom_toTopOf="@+id/activity_main_bottom_navigation"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/salesOverviewTopMenu" />
+        app:layout_constraintTop_toBottomOf="@+id/salesOverviewTopMenu"
```

Comment:
Make sure to test constraint layouts on small screens. Just a reminder so we don't have to apply corrections later on.